openff/interchange/_tests/unit_tests/smirnoff/test_nonbonded.py

```diff
@@ -345,6 +354,8 @@ class TestNAGLChargesPrecedence:
 
     def test_nagl_charges_precedence_over_am1bcc(self, sage_nagl, hexane_diol):
         """Test that NAGLCharges takes precedence over ToolkitAM1BCC."""
+        pytest.importorskip("openff.nagl")
```

Comment:
(not blocking) I forget how pytest decorators work - Could this be made into a `@pytest.requires` and applied to the whole `TestNAGLChargesPrecedence` class? Fine if not, might just make these easier to wrangle/extend in the future.